app/components/page_heading/view.rb

```diff
@@ -0,0 +1,29 @@
+class PageHeading::View < GovukComponent::Base
+  attr_accessor :title, :errors
+  include ActiveModel
+  def initialize(title: "", errors: false)
+    @title = title
+    @errors = errors.present?
+  end
+
+  def build_page_title
+    [build_error + build_title, build_service_name, "GOV.UK"].join(" - ")   
+  end
```

Comment:
Apply's version:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/9227dcb8689bdd9390aecafce7f3b51510cb3189/app/helpers/application_helper.rb#L2

looks like it handles not having a title (which we probably want on the homepage/start page.

Reply:
Should be good for now, it can handle there being no title, in which case it goes to the default of not using it.